pages/models.py

```diff
@@ -183,6 +199,9 @@ class MicrositePage(Page):
     twitter_url = models.URLField(null=True, blank=True)
 
     content_panels = Page.content_panels + [
+            FieldPanel('show_accent_border'),
+            FieldPanel('accent_border_color'),
+            FieldPanel('primary_content_embed_code'),
```

Comment:
I suggest to put the fields at top which are most likely to be edited and accessed regularly. Suggest to put these lower as they are likely to be configured once per page and then probably not needed after that.

Reply:
Oh I actually meant to talk about this during review of model but didn't have time - it makes sense to me to put them in the order they go in the page. Going back and forth editing this model was extremely confusing - I essentially had to control F to find anything because they weren't. Fine if I order the rest of them that way - in the order they're presented on the final page?

Reply:
I think it's better so that the fields which needs to be maintained at top, and the fields which will only be edited once should be below. Within those groups, I think it makes sense to sort by top down matching layout, which I did. But please do take into account the long term maintenance use case where only really the content fields are updated and the logo is probably not going to change much, for example.

Reply:
I think it's a valid point but I will just say it was very, very confusing creating this this page through the CMS because everything was out of order. A user will always have to create this page, so making that first step confusing to make an optional step (updating the page throughout) that may or may not happen doesn't make the most sense to me. In the event of an update a user would know where to scroll in the page to update said content, whereas if they're not ordered in that way a user will not understand the method by which they are ordered.

Reply:
Either way is fine, not a big deal really.